search/src/kernels.cu

```diff
@@ -124,25 +124,25 @@ __global__ void searchFilterImages(int imageCount, int width, int height,
             phiSum += cPhi;
             psiArray[i] = cPsi;
             phiArray[i] = cPhi;
-            if (cPhi == 0.0) {
+            if (cPhi != 0.0) {
```

Comment:
This change looks scary!

If `cPhi` *is* zero, then we divide by it later...

Reply:
Oh. This is a legitimate error that got in somehow. I'm not sure. But I've reverted it.